cmd/spire-agent/cli/healthcheck/healthcheck_windows_test.go

```diff
@@ -0,0 +1,106 @@
+//go:build windows
```

Comment:
Hmm, is it worth trying to arrange this so that we don't have to duplicate all of the tests?